plugins/woocommerce-blocks/tests/e2e/test-data/data/data.ts

```diff
@@ -78,14 +79,14 @@ export const capReviews = [
 	{
```

Comment:
I've sorted this data by the order its created so that its a bit easier to write an assertion that makes sense.